test/xpu_api/utilities/pairs/pairs.spec/ref_assign_g.pass.cpp

```diff
@@ -0,0 +1,114 @@
+// -*- C++ -*-
```

Comment:
I propose to remove this test because this functionality is covered by other tests.